WordPress/Classes/Utility/Analytics/WPAnalyticsEvent.swift

```diff
@@ -444,6 +448,12 @@ import Foundation
             return "category_filter_selected"
         case .categoryFilterDeselected:
             return "category_filter_deselected"
+
+        // User Profile Sheet
+        case .userProfileShown:
+            return "user_profile_shown"
+        case .userProfileSiteShown:
+            return "user_profile_site_shown"
```

Comment:
Would it make sense to call these `liker_profile_*` rather than `user_profile_*`?   I'm thinking that someone looking at tracks might at a glance assume that user_profile referred to the logged in user's own profile.

Reply:
Well, per the User Profile designs, this sheet could be displayed from somewhere other than the liker. So I made it specific to the sheet, not the presenter. That's why I added a source to `user_profile_shown`. I could add a `source` to `user_profile_site_shown` if we like. 

Or, I could change them to `user_profile_sheet_*` if that's better. Which it may be...

Reply:
> So I made it specific to the sheet, not the presenter

Oh I see.  Makes sense.

>  I could change them to user_profile_sheet_*

I think I'm a fan :)

Reply:
Can do. Thanks!